requirements/requirements.common.txt

```diff
@@ -108,7 +108,7 @@ django==4.2.23
     #   insights-analytics-collector
     #   pulpcore
     #   social-auth-app-django
-django-ansible-base[feature-flags,jwt-consumer] @ git+https://github.com/ansible/django-ansible-base@2025.7.1
+django-ansible-base[feature-flags,jwt-consumer] @ git+https://github.com/alancoding/django-ansible-base@remote_rbac
```

Comment:
This one of the things that needs changed before this PR is merged but after dab is merged?